ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -446,7 +446,7 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
       TEST_TYPE: upgrade-conformance
-    workflow: openshift-upgrade-aws
+    workflow: openshift-upgrade-aws-sdn
```

Comment:
was this only wrong in 4.12 (not in 4.11?)

Reply:
(looks like there was no 4.11 equivalent to this job)

Reply:
and, 4.11 would use sdn by default as well.